modules/dnn/CMakeLists.txt

```diff
@@ -148,16 +148,16 @@ ocv_add_perf_tests(${INF_ENGINE_TARGET}
     FILES Include ${perf_hdrs}
 )
 
-ocv_option(${the_module}_PERF_CAFFE "Add performance tests of Caffe framework" OFF)
-ocv_option(${the_module}_PERF_CLCAFFE "Add performance tests of clCaffe framework" OFF)
+ocv_option(OPENCV_DNN_PERF_CAFFE "Add performance tests of Caffe framework" OFF)
+ocv_option(OPENCV_DNN_PERF_CLCAFFE "Add performance tests of clCaffe framework" OFF)
 if(BUILD_PERF_TESTS)
-  if (${the_module}_PERF_CAFFE)
+  if (OPENCV_DNN_PERF_CAFFE)
```

Comment:
Please provide compatibility shim for old variables too:

```
if (OPENCV_DNN_PERF_CAFFE
    OR ${the_module}_PERF_CAFFE  # compatibility for deprecated option
)
```

Or emit warning about removed variables through `ocv_declare_removed_variable` / `ocv_declare_removed_variables` (less compatible, probably should go to the "next" branch only)